getting-started/proc-add-custom-app-file-openshift.adoc

```diff
@@ -30,15 +30,34 @@ data:
 . Go to the *Helm* tab.
 +
 The list of Helm Releases appears on the page.
-. Click the three dots of a Helm release and select *Upgrade*.
-. Add the following values in *Root Schema -> Backstage Chart Schema ->  Backstage Parameters -> Extra App Configuration files to inline into command arguments*:
+. Click the three dots corresponding to a Helm release and select *Upgrade*.
+. Use one of the following views to edit the Helm configuration:
+
+** Using *Form view* 
++
+.. Expand *Root Schema → Backstage chart schema → Backstage parameters → Extra app configuration files to inline into command arguments*. 
+.. Click the *Add Extra app configuration files to inline into command arguments* link.
+.. Enter the value in the following fields:
 +
 --
-* *ConfigMapRef*: `app-config-rhdh`
+* *configMapRef*: `app-config-rhdh`
 * *filename*: `app-config-rhdh.yaml`
 --
-. Click *Upgrade*.
-
+.. Click *Upgrade*.
 
+** Using *YAML view* 
```

Comment:
[This section](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/135/files#diff-2e8740a4b09b45b2f17d1753bd5040018a1b32d267e415ff37f7da0997ff3f3fR29) of PR #135 writes about the Form and YAML views a little differently. At some point, we should ensure we are presenting the Form and YAML configuration instructions consistently across the entire RHDH doc set.

Reply:
Yes, @linfraze. 
What i noticed in the PR#135: if you read line no 29-31, you find lesser information and more of text and redundant phrases. for example, `to configure the {product-short} instance`, same phrase we have repeated in the sub-bullet points. IMO, the way we have introduced form view and YAML view config in this PR is to the point and more catchy and visible.

Reply:
In addition to this, using the YAML code snippet provides better clarity and helps in understanding the correct placement of parameters and values within a long YAML code.